feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Optional } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { OrderEntry } from '@spartacus/core';
+import { CartItemContext } from '@spartacus/storefront';
+import { EMPTY, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { CommonConfiguratorUtilsService } from '../../shared/utils/common-configurator-utils.service';
+import { LineItem } from './configurator-cart-entry-bundle-info.model';
+import { ConfiguratorCartEntryBundleInfoService } from './configurator-cart-entry-bundle-info.service';
+
+@Component({
+  selector: 'cx-configurator-cart-entry-bundle-info',
+  templateUrl: './configurator-cart-entry-bundle-info.component.html',
```

Comment:
better have this: `changeDetection: ChangeDetectionStrategy.OnPush,`

Reply:
picked

Reply:
Done

Reply:
Do you push your changes?

Reply:
Pushed!

Reply:
where the changes?